meta/client/tidbclient/bucket.go

```diff
@@ -266,22 +266,26 @@ func (t *TidbClient) DeleteBucket(bucket Bucket) error {
 }
 
 func (t *TidbClient) UpdateUsage(bucketName string, size int64, tx interface{}) (err error) {
-	var sqlTx *sql.Tx
-	if tx == nil {
-		tx, err = t.Client.Begin()
-
-		defer func() {
-			if err == nil {
-				err = sqlTx.Commit()
-			}
+	if helper.CONFIG.PiggybackUpdateUsage {
```

Comment:
better to make short if-branch appear first
```go
if !helper.CONFIG.PiggybackUpdateUsage {
    return nil
}
// otherwise store usage info to DB
```